concepticondata/conceptlists/Wittmann-1973-200.tsv

```diff
@@ -0,0 +1,201 @@
+Wittmann-1973-200-46	46	1242	FEW	few	pa buku, tigin	pa âpil, kèk	peu de
+Wittmann-1973-200-47	47	1423	FIGHT	to fight	lager	gumê	lutter, se battre
+Wittmann-1973-200-48	48	221	FIRE	fire	dife	difé	feu
+Wittmann-1973-200-49	49	227	FISH	fish	posò	pwésô	poisson
+Wittmann-1973-200-50	50	493	FIVE	five	sêk	sêk	cinq
+Wittmann-1973-200-51	51	1574	FLOAT	to float	flot-e	flóté	flotter
+Wittmann-1973-200-52	52	2003	FLOW	to flow	kul-e	kulé	couler
+Wittmann-1973-200-53	53	239	FLOWER	flower	fler	flè	fleur
+Wittmann-1973-200-54	54	1441	FLY (MOVE THROUGH AIR)	to fly	âvol-e	vólé	voler
+Wittmann-1973-200-55	55	249	FOG	fog	bruyar	bruya	brouillard
+Wittmann-1973-200-56	56	1301	FOOT	foot	lipye	pié	pied
+Wittmann-1973-200-57	57	1500	FOUR	four	kat	kat(r)	quatre
+Wittmann-1973-200-58	58	1431	FREEZE	to freeze	gê fre, kôzele	žélé	geler
+Wittmann-1973-200-59	59	1507	FRUIT	fruit	fri	fwi	fruit
+Wittmann-1973-200-60	60	1447	GIVE	to give	don-e	ba(y)	donner
+Wittmann-1973-200-61	61	1035	GOOD	good	bô	bô	bon
+Wittmann-1973-200-62	62	606	GRASS	grass	lerb	zèb	herbe
+Wittmann-1973-200-63	63	1425	GREEN	green	ver	vèt	vert
+Wittmann-1973-200-64	64	1334	GUTS	guts	trip	trip	tripes, intestins
+Wittmann-1973-200-65	65	1040	HAIR	hair	seve	šivé, pwèl	cheveu(x), poil(s)
+Wittmann-1973-200-66	66	1277	HAND	hand	lamê	mê	main
+Wittmann-1973-200-67	67	1211	HE	he	li	li	lui
+Wittmann-1973-200-68	68	1256	HEAD	head	latet	tèt	tête
+Wittmann-1973-200-69	69	1408	HEAR	to hear	tâd-e	tâdé	entendre
+Wittmann-1973-200-70	70	1223	HEART	heart	leker	kè, čè	cœur
+Wittmann-1973-200-71	71	1210	HEAVY	heavy	lurd	lu	lourd
+Wittmann-1973-200-72	72	136	HERE	here	isi	isit	ici
+Wittmann-1973-200-73	73	353	STRIKE	to hit	bat-e	frapé, tapé	frapper
+Wittmann-1973-200-74	74	1448	HOLD	to hold	tini	kêbé	tenir
+Wittmann-1973-200-75	75	1239	HOW	how	kumâ, ki manyer	kômâ	comment
+Wittmann-1973-200-76	76	1435	HUNT	to hunt	lasas, pus-e	šasé	chasser
+Wittmann-1973-200-77	77	1200	HUSBAND	husband	mari, bonom	mari	mari
+Wittmann-1973-200-78	78	1209	I	I	mo-a	mwê	moi
+Wittmann-1973-200-79	79	617	ICE	ice	laglas	glas	glace
+Wittmann-1973-200-80	80	1459	IF	if	si	si	si
+Wittmann-1973-200-81	81	1460	IN	in	dâ	nâ, lâ	dans
+Wittmann-1973-200-82	82	1417	KILL	to kill	tuy-e	tuyé	tuer
+Wittmann-1973-200-83	83	1410	KNOW (SOMETHING)	to know	kon-e	kôn(ê)	connaître
+Wittmann-1973-200-84	84	624	LAKE	lake	reservoar	(l)étâ	lac
+Wittmann-1973-200-85	85	1355	LAUGH	to laugh	rir	ri	rire
+Wittmann-1973-200-86	86	628	LEAF	leaf	fey	fèy	feuille
+Wittmann-1973-200-87	87	244	LEFT	left (side)	gos	gòš	gauche
+Wittmann-1973-200-88	88	1297	LEG	leg	lazam	žâm	jambe
+Wittmann-1973-200-89	89	1411	LIE (REST)	to lie	alôz-e	kuše, layé	être couché
+Wittmann-1973-200-90	90	1422	BE ALIVE	to live	viv	viv	vivre
+Wittmann-1973-200-91	91	1224	LIVER	liver	lefoa	fwa	foie
+Wittmann-1973-200-92	92	1203	LONG	long	lon	lô(g)	long
+Wittmann-1973-200-93	93	1392	LOUSE	louse	lipu	pu	pou
+Wittmann-1973-200-94	94	2106	MALE PERSON	man-male	zom	nòm, nèg	homme
```

Comment:
Not sure about this one. _homme_ is commonly mapped to MAN.

Reply:
(see comment above) In Swadesh, it's MALE PERSON as well.